### ISSUE TITLE
Error taking the inverse of a matrix

### ISSUE THREAD

**Issue by Maximilian-Stefan-Ernst** [symengine/SymEngine.jl#209]
While precompiling SymEngine, I get the Error messages:
```
WARNING: importing deprecated binding Compat.invokelatest into SymEngine.
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\SymEngine.jl:6
WARNING: importing deprecated binding Compat.LinearAlgebra into SymEngine.
WARNING: importing deprecated binding Compat.Libdl into SymEngine.
```
I am using Julia 1.4.1.

Also, this code

```
using SymEngine
@vars x1 x2 x3 x4

A = [x1 x2
       x3 x4]

inv(A)
```

Gives the Error

```
ERROR: ArgumentError: Object can have no free symbols
Stacktrace:
 [1] N(::SymEngine.BasicType{Val{:Abs}}) at C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\numerics.jl:149
 [2] N(::Basic) at C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\numerics.jl:111
 [3] <(::Basic, ::Basic) at C:\Users\maxim\.julia\packages\SymEngine\RK5Ks\src\mathops.jl:84
 [4] > at .\operators.jl:294 [inlined]
 [5] generic_lufact!(::Array{Basic,2}, ::Val{true}; check::Bool) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\LinearAlgebra\src\lu.jl:146 [6] #lu!#140 at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\LinearAlgebra\src\lu.jl:130 [inlined]
 [7] #lu#142 at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\LinearAlgebra\src\lu.jl:273 [inlined]
 [8] lu at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\LinearAlgebra\src\lu.jl:272 [inlined] (repeats 2 times)
 [9] inv(::Array{Basic,2}) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\LinearAlgebra\src\dense.jl:770
 [10] top-level scope at none:0
```

**Comment by isuruf** [comment]
I'm not sure why it doesn't work with new Julia versions. As a workaround you can do the following.
```
Matrix(inv(SymEngine.CDenseMatrix(A)))
```

**Comment by isuruf** [comment]
Fixed in 0.8.1
